### ISSUE TITLE
About Source-Only Performance

### ISSUE THREAD

**Issue by JialeTao** [a514514772/DISE-Domain-Invariant-Structure-Extraction#5]
Hi, congratulation on the work!  I noticed that you used PASCAL VOC pretrained model  and got source-only performance of 39.8 which is higher than that in AdaptSegNet of 36.6. But in the code of seg_model.py, I see that you still use the pretrained model in AdaptSegNet (the same restore_from). I'm a little confused about this. And is the weights in the google drive you provided pretrained on PASCAL VOC as the paper said? And I wonder if the PASCAL VOC pretrained model is available, thanks.

**Comment by hui-po-wang** [comment]
Hi @JialeTao ,

The strong assumption behind the output-space adaptation (i.e. Tsai et al.) toward good performance is that the label distribution of the source domain should be almost identical to the one of target domain (or vice versa, similar meaning). Given this, we slightly clean the source data to remove figures captured from the perspective of pedestrians (see gta file list in this project). It is reasonable because the target environment aims for autonomous driving and not for autonomous 'robots walking'. Note that this operation was only applied to GTA5.

Additional two changes were also made to the baseline model at training time: (a) increasing the batch size to 2 and (b) conducting random cropping at input, as mentioned in the main manuscript. 
The above two changes thus gave models more information about the inputs. 

Above all, in order to have fair comparison, the results of Tsai et al. (c.f. Seg-map Adaptation in Table.4 of the main manuscript) were obtained by running their code on top of the same baseline, i.e. also with the cleaned GTA5 dataset, the increased batch size and random cropping.

If you are interested, I would be more than happy to see any experiments on analyzing the contributions of these components.

Thanks

**Comment by JialeTao** [comment]
Hi, thanks a lot. I see, you mean you use the same pretrained model but change the batch size, do random cropping and clean the source data, right? If that I'll try these components and find their contribution. I was  confused previously that it said the model is pretrained on  PASCAL VOC dataset in the paper while I remember the model is pretrained on COCO dataset in AdaptSegNet.

**Comment by hui-po-wang** [comment]
Yes. There could be a typo in the paper. I will figure it out and revise our arXiv paper according. :(
